### PR TITLE
Update setting-up-your-phone-system.md

### PR DESCRIPTION
**Pull request by rededwin**
Add links to Teams Phone advanced deployment guide available on Setup.Microsoft.Com and Admin Center

### CODE REVIEW

File: Teams/setting-up-your-phone-system.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ This article introduces the following steps. Each step contains links to more de
 > [!NOTE]
 > Be aware that some steps will differ depending on the PSTN connectivity option you choose. For example, the steps and sequence for phone number management and emergency calling management might differ. These differences are described in detail in the associated articles.
 
+> [!TIP]
+> As a companion to this article, we recommend using the [**Plan and implement your Microsoft Teams Phone deployment setup guide**](https://go.microsoft.com/fwlink/?linkid=2224790) when signed in to the Microsoft 365 Admin Center. This guide will customize your experience based on your environment.  To review best practices without signing in and activating automated setup features, go to the [**M365 Setup portal**](https://go.microsoft.com/fwlink/?linkid=2223356).
```
Comment (JohanFreelancer9) [inline Teams/setting-up-your-phone-system.md:55]:
```suggestion
> As a companion to this article, we recommend using the [Plan and implement your Microsoft Teams Phone deployment setup guide](https://go.microsoft.com/fwlink/?linkid=2224790) when signed in to the Microsoft 365 admin center. This guide will customize your experience based on your environment.  To review best practices without signing in and activating automated setup features, go to the [Microsoft 365 setup portal](https://go.microsoft.com/fwlink/?linkid=2223356).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rededwin, kindly commit the suggested change. Many thanks.

**Comment by rededwin** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
I need to review this content before adding a link to the top of the article, which will redirect people away from the roadmap.  Also, the link listed in lieu of having access to the admin center takes me to a generic 365 setup--instead of one specific to Phone System.  I am closing this PR until I can do further research.

**Comment by CarolynRowe** [comment]
It is probably most appropriate to include this link in the related information section.

**Comment by rededwin** [comment]
We have been working with Serdar to bring awareness and visibility to Advanced Deployment Guides for Teams all up in M365 Admin Center and our unauthenticated Setup.Microsoft.Com portal for all users to access and leverage for accelerated deployment.  Go link for M365 Setup Portal has been fixed and now goes to the Teams Phone Advanced Deployment Guide correctly.  Here is also an [article](https://learn.microsoft.com/en-us/microsoft-365/enterprise/setup-guides-for-microsoft-365?view=o365-worldwide) that explains context and business value of these guides.  This article was selected for relevancy and high page views/visit traffic.

Here is a similar use of this content in [Surface Hub](https://learn.microsoft.com/en-us/surface-hub/first-run-program-surface-hub)

Thank you

@SerdarSoysal

**Comment by CarolynRowe** [comment]
Can you contact me directly please?  I'm responsible for the content/quality of the article, which was also signed off by PM.  I'm not opposed to linking to the content, but I do not want to link to a generic setup site at the top of the article.  We should discuss offline.  I also want to review the content to make sure it is consistent with the content in the article.

**Pull request closed without merging**